docs/tools/plist.md

```diff
@@ -0,0 +1,301 @@
+# Static Analysis PLIST Format Specification
+
+Plist files are used in CodeChecker to store and parse diagnostic 
+information from static analysis tools.
+This documentation explains the structure of plist files generated by 
+CodeChecker, illustrating how they are represented in the code and what 
+they look like on the CodeChecker UI.
+
+## Table of Contents
+* [Plist files in CodeChecker](#plist-files-in-codechecker)
+* [Structure of a .plist File](#structure-of-a-plist-file)
+* [Diagnostic Entry Examples](#diagnostic-entry-examples)
+  * [1. `bug_path_event` — Bug Path Events](#1-bug_path_event--bug-path-events)
+  * [2. `notes` — Notes](#2-notes--notes)
+  * [3. `macro_expansions` — Macro Expansions](#3-macro_expansions--macro-expansions)
+  * [4. `report-annotation` — Report Annotations](#4-report-annotation--report-annotations)
+* [License](#license)
+
+
+## Plist files in CodeChecker
+
+Plist files are produced by the report-converter in CodeChecker from the 
```

Comment:
When mentioning the report-converter, refer to this doc https://github.com/Ericsson/codechecker/blob/master/docs/tools/report-converter.md